### PR TITLE
Added toastify and search filter

### PR DESCRIPTION
**Pull request by sohan9819**
Added the search filters 
Toast feedbacks on user actions

### CODE REVIEW

**Comment by athar05** [comment]
The UI looks professional and fabulous. 

I noticed a few typos.

image

You can change "By Now" to "Buy Now", and "Sign IN" to "Sign In". And also, after the button changes from "Add To Cart" to "Remove From Cart", you can also change the icon to trash icon. 

Rest looks good to merge

File: src/components/CartProductCard.jsx [hunk 1]
```diff
@@ -73,6 +74,7 @@ const CartProductCard = ({ product }) => {
               type: 'Remove from Cart',
```
Comment (athar05) [inline src/components/CartProductCard.jsx:74]: I am just nitpicking here, but the convention would be "REMOVE_FROM_CART". Maybe from the next project you can implement it. 😅

**Review by athar05 — COMMENTED** [review]
Oveall the code looks clean.

File: src/components/CartProductCard.jsx (hunk 1 above)
Comment (sohan9819) [reply src/components/CartProductCard.jsx:74]: Ohh will sure keep that in mind

**Review by rahul31agrawal — APPROVED** [review]
Looks good to merge

**Pull request merged**